ch<!doctype html>
<html>
  <head>
    <!-- Page setup -->
    <meta charset="utf-8">
    <title>Microbial Ecology and Macroevolution Group</title>
    <meta name="description" content="A brief description of your site for search engines">
    <meta name="author" content="Information about the author here">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no"/>
    <link rel="icon" type="image/png" href="favicon.png">
  
    <!-- Stylesheets -->
    <!-- Reset default styles and add support for google fonts -->
    <link href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css" rel="stylesheet" type="text/css" />
    <link href="http://fonts.googleapis.com/css?family=Roboto" rel="stylesheet" type="text/css" />
   
    <!-- Custom styles -->
    <link href="style.css" rel="stylesheet" type="text/css" />

    <!-- jQuery -->
    <script src="https://code.jquery.com/jquery-3.4.1.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>    

    <!-- Want to add Bootstrap? -->
    <!-- Visit: https://getbootstrap.com/docs/4.3/getting-started/introduction/ -->
    
  </head>
  
  <body>

    <header id="header">
      <img src="frog_bacteria_combo.jpg">
      <h1>Microbial Ecology and Macroevolution Group</h1>
      
      <!-- Menu link fragment #id should match a div id. Example: <a href="#home"> links to <div id="home"></div>  -->
      <ul class="main-menu">
        <li><a href="#About">About</a></li>
        <li><a href="#People">People</a></li>
        <li><a href="#Projects">Projects</a></li>
        <li><a href="#Classes">Classes</a></li>
	<li><a href="#Supplemental">Supplemental Material</a></li>  
      </ul>                 
    </header>
   
    <div id="container">
      <div class="inner">
        <div id="content"> 
          
          <div id="About" class="content-region hide">
            <h2>Research Program</h2>
            <p>
              The Microbe-host Ecology and Macroevolution (MEME) group is broadly insterested in pattern, process, and form (PPF) at the interface of ecology and macroevolutionary biology. Specifically, our group seeks to delineate and explain contemporary ecological phenomena in the context of macroevolutionary contraints and adaptations across multiple levels of biological organization. Ecological PPF related to microbe-host (i.e. microbe-metazoan) interactions are of particular interest for their necessary reflection of innovations, constraints, and adaptations over geological time. This formalist-centric (yet functionalist inclusive) paradigm uses microbe-host systems that provide a useful framework for addressing questions at multiple levels of temporal and organizational scale. General areas and specific questions the research group takes interest in are described below. For a more detailed and non-technical overview of my research program, please see my recently posted essay here: 
	      <a href="https://www.extinctblog.org/extinct/2024/11/14/microbe-metazoan-macroevolution">Microbe-metazoan macroevolution: new directions for an old research program</a> 
            </p>
	    <h3>Evolutionary Novelties, Innovations, and their Microbial Constraints</h4>
	    <p>
	    How does the causal role of microbes in: (1) emergence of macroevolutionary novelties and innovations (e.g. speciation/extinction trends, endothermy, adaptive immunity, morphological baupläne) and (2) constraints on adaptation manifest in contemporary ecological interactions of microbes and the PPF of their multicellular organismal hosts?
	    </p>
	    <h3>Levels of Action (canonically as the “levels of selection”)</h4>  
	    <p>
	    Do answers to the above questions vary by level of biological organization (e.g. gene, organism, group, species, clade, community, gaia)?
	    </p>
	    <h3>Methodological, Conceptual, and Tool Development</h4>
	    <p>
	    Answering the above questions require the use of new and emerging methods. Conceptually, this includes theory development and new forms of causal and explanatory reasoning. Technically and methodologically, I have ongoing work addressing tool development. This includes new multi-omics work with preserved and museum specimens, incorporation efforts of historic land use and paleobiology datasets, and utilization of emerging computational technologies (e.g. qubits for de novo assembly problems).
	    </p>
          </div>
          
          <div id="People" class="content-region hide">
            <h2>Joseph Madison, Ph.D. (Principal Investigator)</h2> 
	    <img src="headshot_updated_size.JPG">
            <p>
             Joseph (Joe) Madison is a biologist and principal investigator and is currently an assistant professor at Xavier University of Louisiana and a visiting scientist at the University of Massachusetts Boston. Joe is interested in microbial evolutionary ecology in the context of host-microbe interactions and macroevolutionary phenomena. Joe also works in the history and philosophy of biology.
            </p>
	    <h2>Ahmed El-Desoky (Undergraduate Researcher)</h2>
	    <img src="Ahmed_Photo_resized.png">
            <p>
             Ahmed is a biologist and computer scientist currently in their final year at Xavier University of Louisiana. Ahmed has had the opportunity to conduct research at LSU School of Medicine and Johns Hopkins Whiting School of Engineering in areas at the intersection between biology and data. Currently, Ahmed is interested in exploring the microbiomes of caecilians and sea anemones using statistical modeling and machine learning.
	    </p>
	    <h2>Dyan Oliver (Undergraduate Researcher)</h2>
	    <img src="Dyan_Photo_resized.png">
            <p>
             Dyan is a biology student at Xavier University of Louisiana. Dyan is starting a project that will examine the microbiomes of sea anemones by developing a captive colony for experimental manipulation.
	    </p>
            <h2>Machai Moore (Undergraduate Researcher)</h2>
            <p>
             Machai is a biology student at Xavier University of Louisiana. Machai is involved in the turtle microbiome project.
	    </p>
	    <h2>Mikylla Maghirang (Undergraduate Researcher)</h2>
            <p>
             Mikylla is a biology student at Xavier University of Louisiana. Mikylla is involved in both laboratory and field work, specifically collecting and processing samples for turtle and mangrove microbiome projects.
	    </p>
	    <h2>Penina (Parker) Berman (Undergraduate Researcher)</h2>
            <p>
             Parker is a public health sciences student at Xavier University of Louisiana. Parker is involved in the turtle and urban rat microbiome projects.
	    </p>  
          </div>
          
          <div id="Projects" class="content-region hide">
            <h2>Amphibian Cladal Trends</h2>
            <p>
              This project aims to answer significant historical questions vis-à-vis host-associated microbes in fluid-preserved amphibian specimens. Specifically, it is hypothesized that microbial gut communities will vary in diversity and function both within and between amphibian assemblages over spatiotemporal gradients. Such changes may also precede disease related declines of the associated hosts, such as chytridiomycosis in amphibians. These hypotheses are being tested by utilizing recently developed techniques for isolation and purification of degraded DNA and shotgun metagenomic sequencing for generating host-associated gut metagenome data. Sequence-based taxonomic and functional analysis is being used with a variety of bioinformatics and machine-learning software on high-performance computing resources. By utilizing museum specimens and their associated locality information, I also aim to explicate changes in host-associated microbiomes of amphibian assemblages in response to differential land-use and land-cover (LULC) in the Midwest and Northeast United States from the 1880s to present. Statistical mixed-effects models are currently being implemented for evaluating changes in host-associated microbial metagenomes and will include the integration of publicly available LULC datasets. The project will therefore further my own training in GIS for integration into microbial ecology research, and also promote the mentoring of students from underrepresented groups in using museum collections for answering questions pertaining to microbial ecology and evolution.
            </p>
            <p>
              Publications: 
            </p>
	    <p>
	      <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0291540">Shotgun Metagenomics versus 16S in Amphibian Museum Specimens</a>    
	    </p>
	    <p>  
	      <a href="https://royalsocietypublishing.org/doi/full/10.1098/rspb.2019.1833">Serratia marcescens Effects on Amphibian Chytridiomycosis Resistance</a>  
            </p>
	    <p>
	      <a href="https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2017.00290/full">Anti-Bd Bacteria Characterization from Costa Rica Amphibian Populations</a>
	    </p>
	    <p>
	      <a href="https://www.int-res.com/abstracts/dao/v147/p141-148/">Biosecurity in Amphibian Research</a>    
	    </p> 
	    <p>
	      <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/ece3.3616">Detecting Amphibian Pathogens in Environmental Samples</a>    
	    </p>  
            <h2>Turtle Shell</h2>
            <p>
              The turtle shell project aims to ask eco-evolutionary questions relating to the microbial role in the origin of the turtle shell, to include ongoing work with extant turtle species throughout North and South America. Turtles in particular have been an important historical focal point for questions relating to evolutionary novelties. 
	            Questions being asked in this system include: How do turtle microbiomes change along environmental gradients? How do turtle-associated microbiomes vary among conspecifics at different sites, and to what extent are turtle microbiomes transmitted vertically (which may help clarify questions related to host-microbe determinism and constraints)? And most importantly, how do turtle associated microbiomes mediate evolutionary-developmental trajectories (e.g. carapacial ridge gene expression patterns) and novelties (rib-scapula inversion) in turtles, specifically the emergence of the shell as a morphological novelty.
            </p>
	    <p>
              Publications: 
            </p>
	    <p>
	      <a href="https://meridian.allenpress.com/journal-of-herpetology/article/56/2/258/484849">False Map Turtle Microbiota Variation</a>    
	    </p> 
	    <p>
	      <a href="https://meridian.allenpress.com/copeia/article-abstract/106/4/580/196648/Bacterial-Microbiota-Response-in-Graptemys">False Map Turtle Roundup Exposure and Microbiota</a>    
	    </p> 
	    <p>
	      <a href="moz-extension://a1c74860-db0f-4d1d-a6b4-229cae866d1a/https://www.researchgate.net/profile/Joseph-Madison/publication/331999032_Surveillance_of_ranavirus_in_false_map_turtles_Graptemys_pseudogeographica_along_the_lower_Missouri_River_USA/links/5ca197cb45851506d738ab08/Surveillance-of-ranavirus-in-false-map-turtles-Graptemys-pseudogeographica-along-the-lower-Missouri-River-USA.pdf">Ranavirus Detection in Turtles</a>    
	    </p>   
            <h2>Human-Rat Synanthropy</h2>
            <p>
              I am currently part of the collaborative Boston Urban Rat Study (BURS). This group aims to answer various questions realting to urban rats and human health in Boston, MA. The component of the study of which I am working is specifically interested in rat microbiome and virome associations with different health outcomes in humans and other animal species. Our understanding of interacting ‘omes’ among changing animal communities and their environments is limited and this study will help clarify these unknowns. We currently are characterizing the microbiome and the virome of urban rats to establish a baseline for the species and to understand how the microbiome and virome change temporally and geographically. Other questions of interest include: How different is the rat microbiome from the human microbiome? What viruses are rats carrying and are they zoonotic? How do the rat microbiome and virome change based on habitat type, season, and interaction with other animals, including humans?
            </p>
            <p>
              Publications: 
            </p>
	    <p>
	      <a href="https://apha.confex.com/apha/2022/meetingapi.cgi/Paper/515829?filename=2022_Abstract515829.html&template=Word">Preliminary Results on Zoonosis and Population Structure in Norway Rats</a>    
	    </p>  
          </div>

          <div id="Classes" class="content-region hide">
            <h2>History and Philosophy of Biology (ISE)</h2>
            <p>
              Description: This course will provide an overview of the historical development of various strands of philosophical thought in the biological sciences, with an emphasis on ecology and evolution. This overview will range from ancient proto-naturalistic thinking to contemporary philosophy of biology, to include ongoing debates with broad societal implications. The course will draw on student discussion and critical thinking as a means for analysis of the emergence, intermixing, and fading of these philosophies. By the end of the course students will be able to understand and analyze the historical development of the dominant philosophical paradigms in the biological sciences. Students will also have practice in contextualizing and critiquing the historical and philosophical bases of related social issues such as neo-eugenics and the biodiversity crisis.
            </p>
            <h2>Biodiversity (XULA)</h2>
            <p>
              Description: Survey of biological diversity, basic principles and concepts of evolutionary theory, biological systematics, taxonomy, and ecology.
            </p>
	    <h2>Capstone: Ecology and Evolution of the Microbiome (XULA)</h2>
	    <p>
              Description: In this course students explore the ecology and evolution of the microbiome through a variety of media, and also discuss and present biological data and biological issues pertaining to microbiomes with and to peers. This course marks the culminating experience for Biology majors in which students synthesize, integrate, and apply the knowledge and skills gained in all of their previous coursework.
            </p>
          </div>
         
          <div id="Supplemental" class="content-region hide">
            <h2>Commitment to Diversity and Inclusion</h2> 
            <p>
             Please read about our commitment to diversity and inclusion in the attached PDF file (below).
            </p>
	    <a href="Diversity_Inclusion.pdf" target="_blank">View PDF</a>
          </div>
		
        </div>
      </div>
    </div>

    <footer>  
      MEME (C) 2024
    </footer>
    
    <!-- Load additional JS scripts here -->
    <script type="text/javascript" src="script.js"></script>
    
  </body>
</html>
